src/main/java/com/opams/arch/entities/ProductEntity.java

```diff
@@ -1,8 +1,38 @@
 package com.opams.arch.entities;
 
-import javax.persistence.Entity;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import java.math.BigDecimal;
 
 @Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
 public class ProductEntity {
+
+    @GeneratedValue
+    @Id
+    private long productId;
+
+    @NotNull
+    @Positive
+    private BigDecimal retailPrice;
+
+    private String size;
+
+    private String color;
```

Comment:
Do you wanna have color as string? Or some reference on color table? Or enum?

Which kind of potential interactions you want to perform with color property? Do you wanna query products by some color?